wp-content/plugins/civicrm/civicrm/ext/afform/core/ang/af/afForm.component.js

```
@@ -333,48 +333,53 @@
         }
+
+    // Collection camp start date and end date validation
+    if (ctrl.getFormMeta().name === 'afformCollectionCampIntentDetails'|| 'afformInstitutionCollectionCampIntent') {
+      var startDateValue = $element.find("af-field[name='Collection_Camp_Intent_Details.Start_Date'] .crm-form-date-wrapper input.crm-form-date").val();
+      var endDateValue = $element.find("af-field[name='Collection_Camp_Intent_Details.End_Date'] .crm-form-date-wrapper input.crm-form-date").val();
+      
+      var institutionStartDateValue = $element.find("af-field[name='Institution_Collection_Camp_Intent.Collections_will_start_on_Date_'] .crm-form-date-wrapper input.crm-form-date").val();
+      var institutionEndDateValue = $element.find("af-field[name='Institution_Collection_Camp_Intent.Collections_will_end_on_Date_'] .crm-form-date-wrapper input.crm-form-date").val();
+      
+      if ((startDateValue && endDateValue) || (institutionStartDateValue && institutionEndDateValue)) {
+        var today = new Date();
```

Biome (1.9.4)

[error] Shouldn't redeclare 'today'. Consider to delete it or rename it.

'today' is defined here:


(lint/suspicious/noRedeclare)